_data/team.yml

```diff
@@ -168,6 +168,12 @@
   type: community
   fingerprint: 
 
+- name: Deeplow
```

Comment:
@mfc thanks! Can the name start with lower case ("deeplow"), since it's a username? (like unman or awokd)

Reply:
sure , done!